### ISSUE TITLE
[SCREENREADER]: Facility Locator - Can we improve the alt text on Mapbox maps for JAWS, other screen readers

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#2753]
@1Copenut commented on [Mon Feb 18 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16987)

## Issue
When users focus the Mapbox map to zoom in or out, screen readers hear a long string of text that makes it hard to understand they are hearing the zoom labels read out. JAWS in particular reads out the following:

"Zoom in zoom out Leaflet vertical bar Map data copyright OpenStreetMap contributors, \ CC-BY-SA, \ Imagely copyright Mapbox trademark".

## Environment
* Windows 7 Enterprise Edition
* Internet Explorer 11
* JAWS 2018
* https://staging.va.gov/find-locations/?address=Dallas%2C%20TX&context=Dallas%2C%20Texas%2C%20United%20States&facilityType=all&location=32.7763%2C-96.7969&page=1&serviceType&zoomLevel=9

## Steps to Recreate
1. Enter IP address listed above in browser
2. Start screenreading device listed in Environment
3. Tab through the page (many links) until you are focused on the map on the right. Confirm hearing the string called out above.

## WCAG or Vendor Guidance (optional)
* [Link Purpose (In Context): Understanding SC 2.4.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html)

---
![picture1](https://user-images.githubusercontent.com/934879/52979492-7ce76e00-339b-11e9-9f71-3de5d8703465.png)

---

@mmiddaugh commented on [Tue Oct 22 2019](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16987#issuecomment-545136371)

@brangi , here is one to evaluate for level of effort and usability impact

**Comment by davidmpickett** [comment]
@laflannery is this still relevant?

**Comment by laflannery** [comment]
@davidmpickett This is incredibly old - this issue is no longer relevant but there is a different issue - When I focus on the zoom buttons themselves I do not here any sort of announcement until I activate them.  Then I hear "zooming in" or "zooming out" (using NVDA).  So if I were a non-visual user this would not be helpful because right now I only know that I should be activating something because I can see what I am focusing on.  If might be better/cleaner to close this and make a new issue with the more recent/relevant issue and details - but up to you.
